openshift/datasync-showcase.yml

```diff
@@ -5,10 +5,10 @@ kind: Template
 labels:
   template: ionic-showcase-server
 metadata:
-  name: mobile-developer-services-datasync-demo-server
+  name: datasync-showcase-server
   annotations:
-    openshift.io/display-name: Mobile Developer Services Data Sync Showcase
-    description: Mobile Developer Services Showcase Server connecting to Postgres Database
+    openshift.io/display-name: Data Sync Showcase 
+    description: Template allows to deploy Data Sync Example Showcase application 
```

Comment:
This template contains an example of the Ionic showcase server and a Postgresql database. They are configured to communicate with one another. Graphql allows a user to query the Postgresql database.